README.md

```diff
@@ -65,16 +65,18 @@ on our
 [Matrix](https://app.element.io/#/room/#plasmapy:openastronomy.org)/[Gitter](https://gitter.im/PlasmaPy/Lobby)
 channel. Both of these are the same chat channel; Gitter uses a bridge to link the two.
 
-## [Weekly](https://calendar.google.com/calendar?cid=bzVsb3ZkcW0zaWxsam00ZTlrMDd2cmw5bWdAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) [community meetings](https://meet.jit.si/plasmapy)
+## [Weekly](TO DO: update invite link) [community meetings](https://zoom.us/j/91633383503?pwd=QWNkdHpWeFhrYW1vQy91ODNTVG5Ndz09#success)
```

Comment:
Thanks! Looks good, there is a weird google bug that makes it a bit tricky to copy to one's calendar, but I'm not sure it's fixable from your end.

The bug is: clicking on 'copy to my calendar' allows for a recurring event a day before the actual date (recurring on Monday instead of Tuesday, etc). Clicking instead 'more info' and then 'Copy to my calendar' instead works perfectly fine